CarbonChat-API/build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+description = "CarbonChat-API"
```

Comment:
Whats the point of setting the description to the same thing as the subproject name?

Reply:
Not sure, its what the old gradle file did, the publish task uses it as the artifact

Reply:
I guess it's fine, but personally I would drop it and just use the project name